src/components/ConnectButton/index.tsx

```diff
@@ -19,7 +19,7 @@ let providerName
 const wallets = getSupportedWallets()
 
 export const onboard = Onboard({
-  networkId,
+  networkId: parseInt(networkId, 10),
```

Comment:
The whole point was not to parse it into a number 😂
But I guess we can't get around the Onboard params type.

Reply:
I think we should report this to the onboard.js team

Reply:
I notified the onboard.js team, they are usually quite fast so maybe the PR can wait

Reply:
We can remove the parse once they update the package.